[BUG] cuebe terminate with segmentation fault when it can't connect to kubernetes

## Describe the bug
When I can't reach the kubernetes control plane, cuebe exits with a SIGSEGV:

```shell
cuebe apply -t debug -c release.context
Deploying to https://<redacted>...
E1119 15:14:30.773987   60333 memcache.go:179] couldn't get current server API group list: Get "https://<redacted>/api?timeout=32s": net/http: TLS handshake timeout
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x1f2c1ad]

goroutine 1 [running]:
github.com/loft-orbital/cuebe/internal/kubernetes.(*Release).Deploy(0xc0007e4e00, {0x2931bf0, 0xc0001a2000}, 0x0)
        github.com/loft-orbital/cuebe/internal/kubernetes/release.go:136 +0x60d
github.com/loft-orbital/cuebe/cmd/cuebe/cmd.applyRun(0xc000231ef0)
        github.com/loft-orbital/cuebe/cmd/cuebe/cmd/apply.go:155 +0x23c
github.com/loft-orbital/cuebe/cmd/cuebe/cmd.applyCmd(0xc000377900, {0xc000338240, 0x4, 0x4})
        github.com/loft-orbital/cuebe/cmd/cuebe/cmd/apply.go:77 +0x45
github.com/spf13/cobra.(*Command).execute(0xc000377900, {0xc000338200, 0x4, 0x4})
        github.com/spf13/cobra@v1.2.1/command.go:860 +0x5f8
github.com/spf13/cobra.(*Command).ExecuteC(0x3649a00)
        github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.2.1/command.go:902
github.com/loft-orbital/cuebe/cmd/cuebe/cmd.Execute()
        github.com/loft-orbital/cuebe/cmd/cuebe/cmd/root.go:39 +0x25
main.main()
        ./main.go:21 +0x17
```

## To Reproduce
Get your targeted Kubernetes cluster out of reach, run `cuebe`.

**Expected behavior**
Should terminate gracefully, giving hints of why it could not run properly.

**Actual behavior**
Terminates and should a SIGSEGV error

## Versions
```shell
cuebe version
Version:        dev
Go version:     go1.17.2
Git commit:     ede20df
Built:          Thu Nov 18 14:31:59 2021
OS/Arch:        darwin/amd64%
```

--- a/internal/kubernetes/release.go
+++ b/internal/kubernetes/release.go
@@ -19,6 +19,7 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"os"
 	"sort"
 
 	"cuelang.org/go/cue"
@@ -117,6 +118,8 @@ func (r *Release) Deploy(ctx context.Context, dryrun bool) error {
 				dr, err := dynamicResourceInterfaceFor(obj, rm, dync)
 				if err != nil {
 					failed = append(failed, obj) // Couldn't find any mapping, retrying next loop
+					fmt.Fprintf(os.Stderr, "Could not apply %s: %s, retrying later...\n", printObj(obj), err)
+					continue
 				}
 
 				data, err := obj.MarshalJSON()

